pkgs/ai-functions/src/ai.ts

```diff
@@ -267,15 +267,23 @@ export const list = new Proxy(function () {}, {
         // Create a schema for an array of strings
         const schema = z.array(z.string())
 
+        const systemPrompt = "Respond only with a numbered, markdown ordered list. Each item should be on a new line starting with a number followed by a period."
+        const mergedConfig = {
+          ...config,
+          systemPrompt: config.systemPrompt 
```

Comment:
The config.systemPrompt should just be config.system ... if that's not correct in the current types, please update the types because that is what it should be